application/dash_application/views/pre_exome.py

```diff
@@ -5,15 +5,14 @@
 from . import navbar
 from ..dash_id import init_ids
 from ..plot_builder import generate
+from ..utility import df_manipulation as util
 import plotly.graph_objects as go
 import pandas as pd
-import gsiqcetl.load
-
-# TODO filter down to MiSeq runs only?
-bamqc_cols = gsiqcetl.load.bamqc_columns('v1')
-bamqc = gsiqcetl.load.bamqc('v1').sort_values(by=[bamqc_cols.Sample, bamqc_cols.TotalReads], ascending=False)
-
+from gsiqcetl import QCETLCache
+from gsiqcetl.column import BamQcColumn
+import pinery
 
+# # TODO filter down to MiSeq runs only?
```

Comment:
At the last meeting we decided we wouldn't filter down to MiSeq runs by default, but we may later add a toggle for only displaying MiSeq runs/filtering by instrument model.